exporters/prometheus/src/main/java/io/opentelemetry/exporter/prometheus/PrometheusMetricReader.java

```diff
@@ -26,9 +27,14 @@ public class PrometheusMetricReader implements MetricReader, MultiCollector {
   private volatile CollectionRegistration collectionRegistration = CollectionRegistration.noop();
   private final Otel2PrometheusConverter converter;
 
-  /** See {@link Otel2PrometheusConverter#Otel2PrometheusConverter(boolean)}. */
-  public PrometheusMetricReader(boolean otelScopeEnabled) {
-    this.converter = new Otel2PrometheusConverter(otelScopeEnabled);
+  /** See {@link Otel2PrometheusConverter#Otel2PrometheusConverter(boolean, boolean, Predicate)}. */
+  public PrometheusMetricReader(
+      boolean otelScopeEnabled,
+      boolean addResourceAttributesAsLabels,
+      Predicate<String> allowedResourceAttributesFilter) {
```

Comment:
We're going to need a builder for this.. Can you add a TODO to this? I can take care of it in a followup.
```
TODO: refactor to public static create or builder pattern to align with project style
```